XamarinCommunityToolkit/Effects/Touch/PlatformTouchEffect.macos.cs

```diff
@@ -0,0 +1,188 @@
+﻿using Xamarin.Forms.Platform.MacOS;
+using AppKit;
+using Foundation;
+using Xamarin.Forms;
+using Xamarin.CommunityToolkit.macOS.Effects;
+using Xamarin.CommunityToolkit.Effects;
+
+[assembly: ExportEffect(typeof(PlatformTouchEffect), nameof(TouchEffect))]
+
+namespace Xamarin.CommunityToolkit.macOS.Effects
+{
+	[Preserve(AllMembers = true)]
```

Comment:
can we remove this? Since we want to make sure that the Linker will remove unused code.